src/dvc_render/vega_templates.py

```diff
@@ -612,6 +614,35 @@ class LinearTemplate(Template):
             },
             "color": {"field": "rev", "type": "nominal"},
         },
+        "params": [
+            {
+              "name": "points",
+              "value": 0,
```

Comment:
it means that they are disabled by default? 

the whole reason for their existence is the case when there is a single item on the plot. Vega has this discussion - https://github.com/vega/vega/issues/1495

Plot will look empty be default in that "edge case". Which happens often in our case - the first epoch of training.

Reply:
Sorry, I forgot to push an update to make the default `0.1`. I also pushed an update to fix the legend opacity at 1 instead of having it vary with the point opacity.

It still doesn't look great by default, especially in VS Code. Can we do anything to improve it? I'm not sure this or having the initial point look empty is worse.

VS Code:

![visualization](https://user-images.githubusercontent.com/2308172/214067414-78258a68-fc19-4639-b04d-b26c2cccbb3a.png)

CLI:

![visualization(19)](https://user-images.githubusercontent.com/2308172/214067634-c087d1a1-a03d-49cc-92c0-261f4db7e0c6.png)